### PR TITLE
fix: update og:url for docs

### PR DESCRIPTION
**Pull request by tirthbodawala**
## Motivation
og:url was not updated when visiting doc file, thus improper title was rendered on social shared cards on facebook and twitter
image

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/Docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes, I have and have contributed previously to Docusaurus as well.

### CODE REVIEW

File: v1/website/core/Showcase.js [hunk 1]
```diff
@@ -33,8 +33,4 @@ Showcase.propTypes = {
   users: PropTypes.array.isRequired,
 };
 
-Showcase.defaultProps = {
```
Comment (JoelMarcey) [inline v1/website/core/Showcase.js]: Why did you remove this?

Comment (endiliey) [reply v1/website/core/Showcase.js]: Pls revert this change

**Comment by endiliey** [comment]
Closing due to stale activity. Send a new PR if needed.

Thank you for the attempt though 😉
